### ISSUE TITLE
pretty-hydra: The caret character "^" is not displayed properly

### ISSUE THREAD

**Issue by msladecek** [jerrypnz/major-mode-hydra.el#35]
Example which demonstrates the error (the "^" character in the last line of "Special" breaks the entire last line of the hydra):
```emacs-lisp
(major-mode-hydra-define org-mode
  (:exit nil :quit-key ("q" "<escape>"))
  ("Special"
   (("," org-ctrl-c-ctrl-c "ctrl-c-ctrl-c")
    ("*" org-ctrl-c-star   "ctrl-c-star")
    ("-" org-ctrl-c-minus  "ctrl-c-minus")
    ("." org-priority      "set priority")
    ("^" org-sort          "sort"))
   "Scheduling"
   (("s" org-schedule     "org-schedule")
    ("d" org-deadline     "deadline"))
   "Todo"
   (("T" org-todo       "menu")
    ("H" org-shiftleft  "left")
    ("J" org-shiftdown  "down")
    ("K" org-shiftup    "up")
    ("L" org-shiftright "right"))
   "Insert"
   (("ia" org-attach       "attach")
    ("il" org-insert-link  "link")
    ("if" org-footnote-new "footnote"))
   "Org"
   (("r" org-refile "refile")
    ("R" helm-org-rifle "rifle"))
   "Navigation"
   (("N" org-next-block     "next block")
    ("P" org-previous-block "previous block")
    ("n" org-next-link      "next link")
    ("p" org-previous-link  "previous link")
    ("o" org-open-at-point  "open link"))))
```

![image](https://user-images.githubusercontent.com/7304317/65822618-bb091200-e247-11e9-8a59-4f13741b9431.png)

`*Warnings*` buffer content:
```
Warning (emacs): Unrecognized key: _: sort                              _
Warning (emacs): Unrecognized key: _: right                              _
```

The same example without the `^` key:
```emacs-lisp
(major-mode-hydra-define org-mode
  (:exit nil :quit-key ("q" "<escape>"))
  ("Special"
   (("," org-ctrl-c-ctrl-c "ctrl-c-ctrl-c")
    ("*" org-ctrl-c-star   "ctrl-c-star")
    ("-" org-ctrl-c-minus  "ctrl-c-minus")
    ("." org-priority      "set priority"))
   "Scheduling"
   (("s" org-schedule     "org-schedule")
    ("d" org-deadline     "deadline"))
   "Todo"
   (("T" org-todo       "menu")
    ("H" org-shiftleft  "left")
    ("J" org-shiftdown  "down")
    ("K" org-shiftup    "up")
    ("L" org-shiftright "right"))
   "Insert"
   (("ia" org-attach       "attach")
    ("il" org-insert-link  "link")
    ("if" org-footnote-new "footnote"))
   "Org"
   (("r" org-refile "refile")
    ("R" helm-org-rifle "rifle"))
   "Navigation"
   (("N" org-next-block     "next block")
    ("P" org-previous-block "previous block")
    ("n" org-next-link      "next link")
    ("p" org-previous-link  "previous link")
    ("o" org-open-at-point  "open link"))))
```
![image](https://user-images.githubusercontent.com/7304317/65822633-f0156480-e247-11e9-9feb-599279b0d80f.png)

In regular hydra the `^` character can be displayed in the docstring if it's first escaped with two slashes: `_\\^_`, maybe this mechanism could be used here when generating the docstring?

**Comment by jerrypnz** [comment]
Thanks for reporting this. I've just pushed a fix to `develop` branch https://github.com/jerrypnz/major-mode-hydra.el/pull/36

Could you test it again and see if it works for you? I'll push it to `master` and make it available in melpa later.

**Comment by msladecek** [comment]
Thank you very much, it works like a charm!
![image](https://user-images.githubusercontent.com/7304317/65909582-42ca5a00-e3c9-11e9-924e-0fad61882167.png)

Thanks for the quick response

**Comment by jerrypnz** [comment]
Just merged the change to `master` and released `0.2.2`. Should be available in Melpa soon.
